packages/suite/src/components/suite/Preloader/SuiteLayout/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { useCustomBackends } from 'src/hooks/settings/backends';
+import { useSelector } from 'src/hooks/suite';
+
+export const useEnabledBackends = () => {
+    const enabledNetworks = useSelector(state => state.wallet.settings.enabledNetworks);
+    const customBackends = useCustomBackends();
+    return customBackends.filter(backend => enabledNetworks.includes(backend.coin));
+};
```

Comment:
nit: no need to do it right there, but this hook, as well is `useCustomBackends` should be a selector, as it has no hook logic: effects, memo or refs; only a selector and some data manipulation. Refactoring those hooks is an ongoing process and we have been slowly moving away from this antipattern 🙏

Reply:
Let's do it in another PR after release 🙏